test/unit/test_asyncify.py

```diff
@@ -36,7 +36,12 @@ def test_asyncify_list_bad(self):
             ('--pass-arg=asyncify-whitelist@nonexistent', 'nonexistent'),
             ('--pass-arg=asyncify-blacklist@main', None),
             ('--pass-arg=asyncify-whitelist@main', None),
-            ('--pass-arg=asyncify-whitelist@main', None),
+            ('--pass-arg=asyncify-blacklist@m*n', None),
+            ('--pass-arg=asyncify-whitelist@m*n', None),
+            ('--pass-arg=asyncify-whitelist@main*', None),
+            ('--pass-arg=asyncify-whitelist@*main', None),
+            ('--pass-arg=asyncify-blacklist@non*existent', 'non*existent'),
+            ('--pass-arg=asyncify-whitelist@non*existent', 'non*existent'),
```

Comment:
should we warn about a wildcard pattern that is not matched? maybe it's ok to not warn, as there may be use cases where the user passes in multiple wildcards with the intent that only one match. but, maybe it's safer to always warn?

Reply:
I think so. This was helpful to me this afternoon and this is the only feedback that users have about whether functions were matched (or whether there was a silly typo).